woocommerce/woocommerce-square
# Add ESLint GitHub Action workflow and Fix eslint issues

### All Submissions:



---

### Changes proposed in this Pull Request:
PR adds ESLint GitHub Action workflow to enforce ESLint rules on pull requests and also improves the codebase by resolving issues reported by ESLint.

Closes #119

### Steps to test the changes in this Pull Request:
1. Verify that ESLint Action runs on PR and reports issues properly. (eg: see logs [here](https://github.com/woocommerce/woocommerce-square/actions/runs/9159227000/job/25179164243))
1. Regression testing of the extension

### Changelog entry
> Add - ESLint GitHub Action workflow to enforce ESLint rules on pull requests.
> Dev - Improved codebase by resolving issues reported by ESLint.
